network/src/peer/fuzzing.rs

```diff
@@ -126,12 +125,12 @@ pub fn fuzz(data: &[u8]) {
         // Wait for "remote" to disconnect (we read all data and socket read
         // returns EOF), we read a disconnect request, or we fail to deserialize
         // something.
-        let connection_notifs = connection_notifs_rx.collect::<Vec<_>>().await;
+        let _connection_notifs = connection_notifs_rx.collect::<Vec<_>>().await;
```

Comment:
Minor, but do we need `connection_notifs` to be dropped at the end of the scope or, immediately.  If it doesn't have to be dropped at the end of the scope, we could just replace it with `_`